Extension/browser/firefox/lib/contentPolicy.js

```diff
@@ -129,12 +129,12 @@ var WebRequestHelper = exports.WebRequestHelper = {
 
```

Comment:
And I'm not sure that we can make a cache here, cause the context parameter could be different each time.